tests/phpunit/unit/includes/Protected_Access.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests\Unit;
+
+use ReflectionClass;
+use ReflectionException;
+
+/**
+ * Trait Protected_Access.
+ *
+ * Allows accessing protected methods and properties for testing.
+ *
+ * @internal
+ */
+trait Protected_Access {
```

Comment:
Since this is identical to `Private_Access`, instead of copying this, we can move the class to a shared location.